salt/kallithea-demo/configs/production.ini

```diff
@@ -0,0 +1,604 @@
+################################################################################
+################################################################################
+# Kallithea - Example config                                                   #
+#                                                                              #
+# The %(here)s variable will be replaced with the parent directory of this file#
+################################################################################
+################################################################################
+
+[DEFAULT]
+debug = false
+pdebug = false
+
+################################################################################
+## Uncomment and replace with the address which should receive                ##
+## any error reports after application crash                                  ##
+## Additionally those settings will be used by Kallithea mailing system       ##
+################################################################################
+#email_to = admin@localhost
+#error_email_from = paste_error@localhost
+#app_email_from = kallithea-noreply@localhost
+#error_message =
+#email_prefix = [Kallithea]
+
+#smtp_server = mail.server.com
+#smtp_username =
+#smtp_password =
+#smtp_port =
+#smtp_use_tls = false
+#smtp_use_ssl = true
+## Specify available auth parameters here (e.g. LOGIN PLAIN CRAM-MD5, etc.)
+#smtp_auth =
+
+[server:main]
+## PASTE ##
+#use = egg:Paste#http
+## nr of worker threads to spawn
+#threadpool_workers = 5
+## max request before thread respawn
+#threadpool_max_requests = 10
+## option to use threads of process
+#use_threadpool = true
+
+## WAITRESS ##
+use = egg:waitress#main
```

Comment:
This says it's using ``egg:waitress`` but the init file above says gunicorn, which is it?